SpatialGDK/Extras/schema/core_types.schema

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 package unreal;
 
+type LoadInfo {
+    string path = 1;
+    bool no_load = 2;
```

Comment:
Is there a more precise name than `No load`?

Reply:
`No Load` is used to match the same variable used in Unreal, though I guess we have the chance to make it a bit more descriptive.